src/components/GlobalDataProvider.tsx

```diff
@@ -25,7 +25,8 @@ const GlobalDataProvider = ({ children }: { children: ReactElement }) => {
 
   useEffect(() => {
     const update = async () => {
-      queryClient.resetQueries();
+      await queryClient.cancelQueries();
```

Comment:
why?

Reply:
Added the cancel to cancel any existing queries prior to call to resetQueries. Saw some oddities when switching process IDs (e.g., mainnet to testnet) and this seems to have helped.